io.github.niccokunzmann.python_dhcp_server.yml

```diff
@@ -0,0 +1,87 @@
+id: io.github.niccokunzmann.python_dhcp_server
+# The runtime is the environment that the application will run in, so you can use dependencies it provides
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+# This is the command that will be run when the application is launched
+command: python_dhcp_server
+# These are the permissions that the application needs
+# Read more about finishing here: https://docs.flatpak.org/en/latest/manifests.html#finishing
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --share=ipc
+  - --allow=multiarch
+  - --persist=/app/config
+modules:
+  - name: python_dhcp_server
+    # There are other types of buildsystems, like autotools, cmake, cmake-ninja, simple, qmake
+    buildsystem: simple
+    # see https://docs.flatpak.org/en/latest/manifests.html#modules
+    build-commands:
+      - ./inside.sh
```

Comment:
please put these inline. If it is too long then maybe the upstream repository should have a build system to install everything.

Reply:
Yes, I moved it to upstream.